### PR TITLE
Remove save button from post header on EA forum

### PR DESCRIPTION
**Pull request by Will-Howard**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204610279041423) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by agnesstenlund** [comment]
I don't think we intended to remove the whole save button, just the label `Save` 😮  (but keep the icon)

File: packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx [hunk 1]
```diff
@@ -305,7 +304,7 @@ const PostsPagePostHeader = ({post, answers = [], dialogueResponses = [], toggle
           {!hideMenu &&
             <span className={classes.actions}>
               <AnalyticsContext pageElementContext="tripleDotMenu">
-                <PostActionsButton post={post} includeBookmark={!isEAForum} />
```
Comment (jpaddison3) [inline packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx:308]: This one definitely seems wrong?

File: packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx [hunk 2]
```diff
@@ -282,7 +282,6 @@ const PostsPagePostHeader = ({post, answers = [], dialogueResponses = [], toggle
           {isEAForum ? <LWTooltip title={postGetCommentCountStr(post, commentCount)}>
             {commentCountNode}
           </LWTooltip> : commentCountNode}
-          {isEAForum && <BookmarkButton post={post} variant='iconWithText' />}
```
Comment (jpaddison3) [inline packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx:285]: Agree that this was not the intended behavior, and that we should be looking to remove the text instead.

**Review by jpaddison3 — CHANGES_REQUESTED** [review]

**Comment by Will-Howard** [comment]
> I don't think we intended to remove the whole save button, just the label Save

My bad, I'll change this

**Review by oetherington — APPROVED** [review]
👍 Works nicely.

A couple of things that would be nice though:

- There's no hover style - probably we should have the same hover style here that we have for the comments button next to it
- The tooltip placement should probably match the comments button too (`BookmarksButton` takes a placement prop so this should be easy)

**Pull request merged**